src/engine/enginebuffer.cpp

```diff
@@ -59,6 +59,8 @@
 
 #include "trackinfoobject.h"
 
+const double kLinearScalerElipsis = 1.00058; // changes < 1 cent allows a linear scaler
```

Comment:
this is a really stupid request, so feel free to ignore it:  can you write out the math you used to get this value?